### ISSUE TITLE
the accuracy of real-time recognition was extremely low！

### ISSUE THREAD

**Issue by Ricky-Sun** [beingaryan/Automated-Sign-To-Speech-Conversion#1]
Hello, when I was working on this project, the results of training and verification were very good, but the accuracy of real-time recognition was extremely low. Do you know the reason for this? Looking forward to your reply~

**Comment by beingaryan** [comment]
Maybe you must have trained the model for a good number of epochs, like - 99% in the training set too, Try using early stopping and 85-88% as the threshold. As here there are 26+1=27 classes, you need to be very accurate while testing(if accuracy is 99%- it includes exact gesture and exact plane background too) for classification.
